### PR TITLE
Feature/nav

### PR DESCRIPTION
**Pull request by edwartexe**

### CODE REVIEW

File: assets/scripts/index.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        settings: {
+          slidesToShow: 1,
+          slidesToScroll: 1,
+          initialSlide:1,
+        }
+      }
+    ]
+  });
+
+});
```
Comment (JesuHrz) [inline assets/scripts/index.js:126]: Please, always add a new line at the end of the file
@edwartexe

File: assets/svg/arrow.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="11" height="21" viewBox="0 0 11 21" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (JesuHrz) [inline assets/svg/arrow.svg:1]: These SVGs are not optimized, you can use these app
For SVG: https://imageoptim.com/mac
For PNG, JPEG and among others: https://tinypng.com/
@edwartexe

File: index.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    </div>
+  </header>
+
+
+
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js" ></script>
+  <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.min.js"></script>
+  <script type="text/javascript" src="./assets/scripts/index.js" defer></script>
+</body>
+</html>
```
Comment (JesuHrz) [inline index.html:54]: Please, always add a new line at the end of the file
@edwartexe

File: index.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+          <li><a href="#work">Our Work</a><span></span></li>
+          <li><a href="#blog">Blog</a><span></span></li>
+          <li><a href="#career">Careers</a><span></span></li>
+          <li class="nav__contact"><a href="#contact">Contact</a></li>
+        </ul>
+      </nav>
+      <a href="#contact" class="nav__phone">
+        <img class="nav__phone-image" src="./assets/svg/phone-green.svg" alt="telefono"/>
+        <span class="nav__tooltip">300 00 00 000</span>
+      </a>
```
Comment (JesuHrz) [inline index.html:44]: This element must be used to make a call, you can use this attribute

```html
<a href="tel:+13000000000" class="nav__phone">
  <img class="nav__phone-image" src="./assets/svg/phone-green.svg" alt="telefono"/>
  <span class="nav__tooltip">300 00 00 000</span>
</a>
```

@edwartexe

File: index.html [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+  <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.css"/>
+  <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick-theme.css"/>
+  <link rel="stylesheet" href="./assets/styles/styles.css"/>
+</head>
+<body>
+
+  <header class="nav">
+    <div class="container flex flex--between">
+      <button class="nav__menu-toggle display-none-lg js-nav-toggle">
```
Comment (JesuHrz) [inline index.html:19]: This class must be a modifier class or a global class like this: `hide` and this class has to hide on some breakproints you can add it modifier class @edwartexe

File: index.html [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+        <span></span>
+        <span></span>
+      </button>
+      <picture class="nav__logo">
+        <source media="(min-width:834px)" srcset="./assets/svg/logo_big.svg">
+        <source media="(min-width:0px)" srcset="./assets/svg/logo_mini_green.svg">
+        <img src="./assets/svg/logo_mini_green.svg" alt="Koombea">
+      </picture>
+      <nav class="nav__menu">
+        <h2 class="display-none">Site Navigation</h2>
```
Comment (JesuHrz) [inline index.html:30]: This class must be a modifier class or a global class like this: `hide` @edwartexe

File: assets/styles/styles.css [hunk 7]
```diff
@@ -0,0 +1,503 @@
...
+
+.title-tertiary {
+  font-weight: bold;
+  font-size: 1.25em;
+  line-height: 120%;
+  letter-spacing: -0.3333333333px;
+  color: var(--main-title-clr);
+}
+
+.text-center {
```
Comment (JesuHrz) [inline assets/styles/styles.css:83]: This is a modifier class and must be written this way: `.text--center` @edwartexe

File: assets/styles/styles.css [hunk 8]
```diff
@@ -0,0 +1,503 @@
...
+
+.flex--center {
+  justify-content: center;
+}
+
+.flex-row-reverse {
+  flex-direction: row-reverse;
+}
+
+.width100 {
```
Comment (JesuHrz) [inline assets/styles/styles.css:114]: This class must be better named, please change it @edwartexe

File: assets/styles/styles.css [hunk 9]
```diff
@@ -0,0 +1,503 @@
...
+.btn-primary:hover {
+  color: var(--white);
+}
+
+.btn-secondary {
+  background-color: var(--light-green);
+}
+.btn-secondary:hover {
+  background-color: var(--green);
+}
```
Comment (JesuHrz) [inline assets/styles/styles.css:160]: Theses classes should be modifier classes like this: @edwartexe 

```css
.btn--secondary {
  background-color: var(--light-green);
}
.btn--secondary:hover {
  background-color: var(--green);
}
```

**Review by JesuHrz — CHANGES_REQUESTED** [review]

File: index.html (hunk 4 above)
Comment (edwartexe) [reply index.html:44]: thanks, i didnt know that!

File: assets/scripts/index.js (hunk 1 above)
Comment (edwartexe) [reply assets/scripts/index.js:126]: why is that?

File: index.html (hunk 5 above)
Comment (edwartexe) [reply index.html:19]: if you mean "display-none-lg" thats exactly what it is
its a generic class that only hides elements in the large media query

File: assets/styles/styles.css (hunk 8 above)
Comment (edwartexe) [reply assets/styles/styles.css:114]: something like .width--100 or .width-full?

File: index.html (hunk 5 above)
Comment (JesuHrz) [reply index.html]: Ye, this class can be named `hide` and you can add it some modifier if you want it, for example:
`hide--only-desktop`, `hide--only-mobile` and something like that @edwartexe

File: assets/styles/styles.css (hunk 8 above)
Comment (JesuHrz) [reply assets/styles/styles.css]: `.width-full` should be good @edwartexe

File: assets/scripts/index.js (hunk 1 above)
Comment (JesuHrz) [reply assets/scripts/index.js]: To avoid the warning of github @edwartexe

**Pull request closed without merging**